source/server/spongineer/properties.rst

```diff
@@ -0,0 +1,66 @@
+the launch command may look like this: ``java -Dsponge.shouldFireAll=true -Dsponge.filter.debug=true -jar forge.jar`` .
+
+.. note::
+   The example above is a simple launch command. Your launch command can, and probably will, have other options.
+
+Properties Available
+====================
+
++------------------------------------+---------------------------------------------------------------------------------+
+| Property                           | Description                                                                     |
++====================================+=================================================================================+
+|                                    | Forces all boolean "performance" checks for common SpongeAPI events to be       |  
+| **sponge.shouldFireAll**           | forcibly created and fired, regardless whether there is a registered event      |
+|                                    | listener for the desired event (Sponge will normally not construct and throw a  |
+| (boolean, default:false)           | ``ChangeBlockEvent`` if there's no listeners for that event or it's sub         |
+|                                    | events).                                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.debugShouldFire**         | Logs changes to the ``ShouldFire`` flags when event listeners are               |  
+|                                    | registered/unregistered causing one of the boolean flags to be flipped (whether |
+| (boolean, default: false)          | it's going from ``true`` to ``false`` or ``false`` to ``true``).                |
++------------------------------------+---------------------------------------------------------------------------------+
+| **timings.bypassMax**              | Allows for bypassing the max number of frames to send to the Aikar's timing     |   
+|                                    | API. Setting this will not permit bypassing the limit unless the API is         |  
+| (any value that exists)            | configured to permit it.                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Adds a StackTrace to each ``StackFrame`` for when either the Sponge             |
+| **sponge.debugcauseframes**        | implementation or a plugin potentially corrupts the ``CauseStackManager`` 's    |
+|                                    | ``StackFrame`` s, whether it's a frame being popped out of order, or popping    |
+| (boolean, default: false)          | an already popped frame. This is useful to find out where the frame was         | 
+|                                    | created.                                                                        |
```

Comment:
Has the potential to increase performance costs due to the stacktrace creation for every frame in the duration of the game's lifecycle.

Reply:
I chose a warning block over additional bold-face text in the table. Let me know if adding to the table was the original intent.